packages/core-api/src/apis/implementations/auth/github/GithubAuth.ts

```diff
@@ -84,7 +87,7 @@ class GithubAuth implements OAuthApi, SessionApi {
 
     const authSessionStore = new AuthSessionStore<GithubSession>({
       manager: sessionManager,
-      storageKey: 'githubSession',
+      storageKey: storageKey,
```

Comment:
Thinking we could just use the `provider.id` here instead. Maps nicely to the uniqueness required in the rest of the system

Reply:
To be clear, something that includes `provider.id`, not just that by itself. Perhaps `authSession:${provider.id}`?

Reply:
okay, how about `${provider.id}Session`?